.github/workflows/spm.yml

```diff
@@ -23,33 +23,47 @@ jobs:
         if: ${{ github.event_name == 'schedule' }}
         run: echo '::set-output name=WAS_CHANGED::'$(test -n "$(git log --format=%H --since='24 hours ago')" && echo 'true' || echo 'false')
 
+  spm-package-resolved:
+    runs-on: macos-14
+    outputs:
+      package_resolved_hash: ${{ steps.cache.outputs.cache-primary-key }}
```

Comment:
This output is unfortunately empty (see my comments in Chat).

Reply:
I found that the `cache` step produced no outputs at all. You can verify by adding a step that `echo`s it in this job:
```
- name: Print outputs
  run: |
    echo "cache-primary-key: " "${{ steps.cache.outputs.cache-primary-key }}"
    echo "All Outputs: " "${{ toJSON(steps.cache.outputs) }}"
```

Or in a dependent job below:
```
- name: Print outputs
  run: |
    echo "package_resolved_hash: " "${{ needs.spm-package-resolved.outputs.package_resolved_hash }}"
    echo "All Outputs: " "${{ toJSON(needs.spm-package-resolved.outputs) }}"
```

Reply:
Thanks! Updated based on https://github.com/firebase/firebase-ios-sdk/pull/13687